hyp3_autorift/process.py

```diff
@@ -87,8 +88,18 @@ def get_s2_metadata(scene_name):
     }
     response = requests.post(S2_SEARCH_URL, json=payload)
     response.raise_for_status()
+
     if not response.json().get('numberReturned'):
-        raise ValueError(f'Scene could not be found: {scene_name}')
+        metadata_dir = Path(__file__).parent / 'metadata' / 's2_metadata.zip'
+        with tempfile.TemporaryDirectory() as tmp_metadata_dir:
+            shutil.unpack_archive(metadata_dir, tmp_metadata_dir)
+            tmp_metadata_path = tmp_metadata_dir / 's2_metadata.json'
+            with open(tmp_metadata_path) as f:
+                s2_metadata = json.load(f)
```

Comment:
I think this can be simplified to:
```suggestion
        metadata_dir = Path(__file__).parent / 'metadata' / 's2_metadata.zip'
        with ZipFile(metadata_dir) as zf:
            with zf.open('s2_metadata.json') as f:
                s2_metadata = json.load(f)

```

Reply:
For the above, you'll want to import `from zipfile import ZipFile` instead of `tempfile`